client/packages.go

```diff
@@ -32,37 +32,37 @@ import (
 
 // Snap holds the data for a snap as obtained from snapd.
 type Snap struct {
-	ID               string        `json:"id"`
-	Title            string        `json:"title,omitempty"`
-	Summary          string        `json:"summary"`
-	Description      string        `json:"description"`
-	DownloadSize     int64         `json:"download-size,omitempty"`
+	ID               string        `json:"id" help:"The unique snap-id"`
+	Title            string        `json:"title,omitempty" help:"The human-readable name"`
+	Summary          string        `json:"summary" help:"The short summary"`
+	Description      string        `json:"description" help:"The multi-line description"`
+	DownloadSize     int64         `json:"download-size,omitempty" help:"The download size"`
 	Icon             string        `json:"icon,omitempty"`
-	InstalledSize    int64         `json:"installed-size,omitempty"`
-	InstallDate      time.Time     `json:"install-date,omitempty"`
-	Name             string        `json:"name"`
-	Developer        string        `json:"developer"`
-	Status           string        `json:"status"`
-	Type             string        `json:"type"`
-	Base             string        `json:"base,omitempty"`
-	Version          string        `json:"version"`
-	Channel          string        `json:"channel"`
-	TrackingChannel  string        `json:"tracking-channel,omitempty"`
+	InstalledSize    int64         `json:"installed-size,omitempty" help:"The installed size in byte"`
+	InstallDate      time.Time     `json:"install-date,omitempty" help:"The date of installation (or empty)"`
+	Name             string        `json:"name" help:"The human readable name of the snap"`
+	Developer        string        `json:"developer" help:"The name of the publisher"`
```

Comment:
Can you please add a note explaining Developer vs Publisher wording.

Reply:
I think we are doing confusing things here as develoepr is actually the publisher. I will remove the help on this for now and prepare a PR that adds a "Publisher" to the client API that is the publisher (and then we can deprecate or fix developer).